packages/components/src/internal/components/productnavigation/ProductNavigationMenu.tsx

```diff
@@ -97,36 +94,33 @@ export const ProductNavigationMenuImpl: FC<ProductNavigationMenuImplProps> = mem
         return <Alert>{error}</Alert>;
     }
 
-    if (!products || !projects || !tabs) {
+    if (!products || !tabs) {
         return <LoadingSpinner wrapperClassName="product-navigation-loading-item" />;
     }
 
     const selectedProduct = getSelectedProduct(products, selectedProductId);
-    const productProjects = selectedProduct ? productProjectMap[selectedProduct.productId] : undefined;
+    const selectedProject = selectedProduct ? new Container(getServerContext().container) : undefined;
```

Comment:
is this `selectedProject` still needed? it is a little hard to tell from this PR diff, but I assume the menu doesn't care about a selected project anymore.

Reply:
We do need the current container, but "selected" is no longer appropriate.